src/views/layouts/example.njk

```diff
@@ -6,12 +6,12 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>{{ title }} | Example | Design System | ONS — Digital Publishing </title>
         <link rel="shortcut icon" href="/assets/images/favicon.ico" type="image/x-icon">
-        <link rel="stylesheet" href="/assets/css/tailwind.css">
         <link rel="stylesheet" href="/assets/css/main.css">
         <meta name="robots" content="noindex, nofollow">
+        <style>main { padding: 1rem; }</style>
```

Comment:
Tailwind in development is about 2.4MB. Unused styles in prod get purged but it's still multiple requests being made if a component has a number of examples/variants in the documentation. Removed the request altogether and replaced with style tags for the specific styles used